src/users/entities/user.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { Column, DataType, Model, Table } from 'sequelize-typescript';
+
+@Table
+export class User extends Model {
+  @Column({
+    type: DataType.UUID,
+    primaryKey: true,
+  })
+  id: string;
+
+  @Column({
+    type: DataType.STRING,
+    allowNull: false,
+  })
+  login: string;
+
+  @Column({
+    type: DataType.STRING,
+    allowNull: false,
+  })
+  password: string;
+
+  @Column({
+    type: DataType.INTEGER,
+    allowNull: false,
+  })
+  age: number;
+
+  @Column({
+    type: DataType.BOOLEAN,
+    allowNull: false,
+    defaultValue: false,
+  })
+  isDeleted: boolean;
+}
```

Comment:
I think it should be a model, not entity

Reply:
complete